src/System.Xml.ReaderWriter/src/System/Xml/AsyncHelper.cs

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace System.Xml
+{
+    internal static class AsyncHelper
+    {
+        public static readonly Task DoneTask = Task.FromResult(true);
```

Comment:
Vote from the peanut gallery with my $0.02.  Everything in a PR should be reviewed, there is no guarantee that 'clean up' will ever happen.  If its not meant to be included, don't submit it.